python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -325,6 +326,25 @@ def log_compute_logs(self, log: DagsterLogManager, run_id: str, step_key: str) -
                 f" {step_key}. Check the databricks console for more info."
             )
 
+    def get_databricks_run_dagster_event(
+        self, context: StepExecutionContext, databricks_run_id: int
+    ) -> DagsterEvent:
+        metadata = {"step_key": context.node_handle.name}
+        run = self.databricks_runner.client.workspace_client.jobs.get_run(
+            databricks_run_id
+        ).as_dict()
+        run_page_url = run.get("run_page_url")
+        if run_page_url:
+            metadata["databricks_run_url"] = MetadataValue.url(run_page_url)
```

Comment:
pyright was being annoying here because it assumes `metadata` is a Dict[str, str] rather than Dict[str, RawMetadataValue]. Just pushed a change to this PR to fix that.

Reply:
Ah, thank you 🙌

Reply:
I squashed the commits for posterity.